crates/flox-rust-sdk/src/models/environment/path_environment.rs

```diff
@@ -23,20 +24,33 @@ use super::{
     PathPointer,
     DOT_FLOX,
     ENVIRONMENT_POINTER_FILENAME,
-    MANIFEST_FILENAME,
 };
 use crate::environment::NIX_BIN;
 use crate::flox::Flox;
 use crate::models::environment::{
+    resolve_manifest_to_lockfile,
+    PkgDbError,
     BUILD_ENV_BIN,
     CATALOG_JSON,
     ENV_FROM_LOCKFILE_PATH,
-    PATH_ENV_GCROOTS_DIR,
 };
 use crate::models::environment_ref::{EnvironmentName, EnvironmentRef};
 use crate::models::manifest::{insert_packages, remove_packages};
-
-const ENVIRONMENT_DIR_NAME: &'_ str = "env";
+use crate::models::search::PKGDB_BIN;
+
+pub const MANIFEST_FILENAME: &str = "manifest.toml";
+pub const LOCKFILE_FILENAME: &str = "manifest.lock";
+pub const PATH_ENV_GCROOTS_DIR_NAME: &str = "run";
+pub const ENVIRONMENT_DIR_NAME: &'_ str = "env";
+
+// The directory structure for a path environment looks like this:
+// .flox/
+//     ENVIRONMENT_POINTER_FILENAME
+//     ENVIRONMENT_DIR_NAME/
+//         MANIFEST_FILENAME
+//         LOCKFILE_FILENAME
+//     PATH_ENV_GCROOTS_DIR_NAME/
+//         $system.$name (out link)
```

Comment:
suggestion (nb): Move to top as module level doc

```suggestion
!// The directory structure for a path environment looks like this:
!// .flox/
!//     ENVIRONMENT_POINTER_FILENAME
!//     ENVIRONMENT_DIR_NAME/
!//         MANIFEST_FILENAME
!//         LOCKFILE_FILENAME
!//     PATH_ENV_GCROOTS_DIR_NAME/
!//         $system.$name (out link)
```